pep-8016.rst

```diff
@@ -118,13 +121,15 @@ Electing the council
 
 A council election consists of two phases:
 
-* Phase 1: Candidates advertise their interest in serving. Only core
-  team members may be candidates.
+* Phase 1: Candidates advertise their interest in serving. Candidates
+  must be nominated by a core team member. Self-nominations are
+  allowed.
 
 * Phase 2: Each core team member can vote for zero to five of the
   candidates. Voting is performed anonymously. Candidates are ranked
-  by the total number of votes they receive. In case of a tie, the
-  candidate who joined the core team earlier wins.
+  by the total number of votes they receive. If a tie occurs, it may
+  be resolved by mutual agreement among the candidates, or else the
+  winner will be chosen at random.
```

Comment:
ah, thanks for fixing this corner case (two candidates who became core devs "at the same time") ;-)